### DISCUSSION TITLE
Spider income earning dropped

### DISCUSSION THREAD

**Discussion by OlivierGaland** [OlivierGaland/CashFactory#20] [General]
Hello,

For information, it seems my earnings dropped to zero on spider income since 29 MAR, so 8 days ago. I forced an ip change with my connection and it is still in the same state. Note I'm using it since 1st JAN. And I made around $2 in 3 monthes ... with a $20 payout 

Please share any comment on this app (especially if you are currently earning something with and for those who are close to 3 months usage if the traffic revenue suddenly dropped after that delay.

So far i'm wondering if it is either a scam or they dropped the dev. I will let it run more a few more weeks then update the infos on it (or remove it from the package).

Olivier.

![image](https://user-images.githubusercontent.com/26048157/162293545-6125bbb2-6965-4cce-a623-c431d2a2c7b7.png)

**Comment by Bidmaster777** [comment]
Hi Olivier, I'm still making some money of it. Not much, but some... see below ;-)
![image](https://user-images.githubusercontent.com/29657091/165734363-9d412cf3-943d-415b-93b8-49d217484819.png)
![image](https://user-images.githubusercontent.com/29657091/165734416-cd821f97-a0b5-4422-8652-143be7aaa697.png)

**Comment by RocketDevil** [comment]
Any news on that App?
I got  0.003cent since last year in it, so i guess its dead.. and can be deleted i guess?
